source/runtime_gui.cpp

```diff
@@ -1672,6 +1672,9 @@ void reshade::runtime::draw_gui_settings()
 		}
 
 		modified |= imgui::directory_input_box("Screenshot path", _screenshot_path, _file_selection_path);
+		modified |= imgui::directory_input_box("Screenshot sound", _screenshot_playsound_path, _file_selection_path);
```

Comment:
This should probably be `imgui::file_input_box("Screenshot sound", "sound.wav", _screenshot_sound_path, _file_selection_path, { L".wav" });`, otherwise can't select an audio file.